libecho/examples/RawConnection/main.cc

```diff
@@ -75,7 +75,7 @@ int main(int argc, char *argv[]) {
 
         printf("Receiving %d bytes. Low frequency: %d, high frequency: %d, window size: %d, mag limit: %d\n", bytes,
                freq1, freq2, winsize, magLim);
-        auto connection = EchoRawConnection::getBitEchoRawConnection(winsize, freq1, freq2, freq1, freq2, magLim);
+        auto connection = EchoRawConnection::getBitEchoRawConnection(winsize, freq1, freq2, freq1, freq2, 0, magLim);
```

Comment:
Czy tu ma zawsze być 0 czy jednak lepiej żeby było to ustawiane w parametrze?

Reply:
na razie 0